charts/virtaitech-orion-vgpu/v1.0.0/values.yaml

```diff
@@ -0,0 +1,95 @@
+image:
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
```

Comment:
The `imagePullSecrets` is defined as an array but used as a string? I think it's better to keep consistent with questions.

Reply:
Changed `imagePullSecrets: []` to `imagePullSecrets: `